.github/workflows/python-packaging.yml

```diff
@@ -57,7 +57,7 @@ jobs:
             os: ubuntu-20.04
             wheel-name: manylinux2014
           - cibw_build: macosx_x86_64
-            os: macos-12
+            os: macos-latest
```

Comment:
is there a reason we don't also move to `ubuntu-latest`?

Reply:
Great idea @bkmartinjr -- happy to try this as a follow-on PR! :)

Reply:
#3186